src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+
+interface Props {
+  isError: boolean,
+  errorText: string,
+  onSetIsError: (isError: boolean) => void,
+}
+export const ErrorNotification: React.FC<Props> = React.memo(({
+  isError,
+  onSetIsError,
+  errorText,
+}) => {
+  let timer: NodeJS.Timer;
+
+  useEffect(() => {
+    if (isError) {
+      timer = setTimeout(() => onSetIsError(false), 3000);
+    }
+  }, [isError]);
```

Comment:
this functional is really good. But it applies to state in App, so I would recommend you to move it to one function that handles error